apps/oc_chef_wm/itest/oc_chef_wm_keys_SUITE.erl

```diff
@@ -248,25 +438,47 @@ http_keys_request(Method, client, Requestor, Body) ->
                            {"accept", "application/json"},
                            {"content-type", "application/json"}], Method, Body).
 
-% Some helpers to keep noise out of the tests...
+http_named_key_request(Method, Type, Requestor, Name) ->
+    http_named_keys_request(Method, Type, Requestor, Name, <<>>).
 
-make_org() ->
-    Org = chef_object:new_record(oc_chef_organization, nil, ?ORG_AUTHZ_ID,
-                                 {[{<<"name">>, ?ORG_NAME}, {<<"full_name">>, ?ORG_NAME}]}),
-    chef_db:create(Org, context(), ?ORG_AUTHZ_ID).
+http_named_keys_request(Method, user, Requestor, Name, Body) ->
+    Url = "http://localhost:8000/users/user1/keys/" ++ Name,
+    ibrowse:send_req(Url, [{"x-ops-userid", binary_to_list(Requestor)},
+                           {"accept", "application/json"},
+                           {"content-type", "application/json"}], Method, Body);
+http_named_keys_request(Method, client, Requestor, Name, Body) ->
+    Url = "http://localhost:8000/organizations/testorg/clients/client1/keys/" ++ Name,
+    ibrowse:send_req(Url, [{"x-ops-userid", binary_to_list(Requestor)},
+                           {"accept", "application/json"},
+                           {"content-type", "application/json"}], Method, Body).
+
+% Some helpers to keep noise out of the tests...
+make_org(OrgName, OrgAuthzId) ->
+    Org = chef_object:new_record(oc_chef_organization, nil, OrgAuthzId,
+                                 {[{<<"name">>, OrgName}, {<<"full_name">>, OrgName}]}),
+    chef_db:create(Org, context(), OrgAuthzId).
 
 make_client(Config, Name) ->
+    make_client(Config, Name, ?CLIENT_AUTHZ_ID).
+
+make_client(Config, Name, Authzid) ->
     OrgId = proplists:get_value(org_id, Config),
+    make_client(Config, Name, Authzid, OrgId).
+
+make_client(Config, Name, Authzid, OrgId) ->
```

Comment:
Not sure we need to split this out for authzid - authzid is always ignored in these tests because authz is  mocked out, so it's safe to always use the same for  every test.  (For that matter, we don't really need different authz IDs for different objects, now that I think on it some... ).  Too, we're not using either of the new AUTHZ2_ID values.